###############################################################################
# Copyright (c) 2016, Robopoly Development Team
# All rights reserved.
#
# Redistribution and use in source and binary forms, with or without
# modification, are permitted provided that the following conditions are met:
#
# 1. Redistributions of source code must retain the above copyright notice,
# this list of conditions and the following disclaimer.
#
# 2. Redistributions in binary form must reproduce the above copyright notice,
# this list of conditions and the following disclaimer in the documentation
# and/or other materials provided with the distribution.
#
# 3. Neither the name of the copyright holder nor the names of its contributors
# may be used to endorse or promote products derived from this software without
# specific prior written permission.
#
# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
# POSSIBILITY OF SUCH DAMAGE.
##############################################################################

#######################################################################################
#
# Title:       PantoDraw Project
# File:        mainwindow.py
# Date:        2016-05-31
# Author:      Paul-Edouard Sarlin
# Website:	   https://github.com/Robopoly/PantoDraw
#
# Description: Graphics for the GUI main window. Generated from file
#              'mainwindow.ui' using PYQt4 UI code generator (v. 4.9.3).
#
#######################################################################################

from PyQt4 import QtCore, QtGui

try:
    _fromUtf8 = QtCore.QString.fromUtf8
except AttributeError:
    _fromUtf8 = lambda s: s

class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName(_fromUtf8("MainWindow"))
        MainWindow.resize(984, 645)
        MainWindow.setWindowTitle(_fromUtf8("PantoDraw"))
        self.horizontalLayout = QtGui.QHBoxLayout(MainWindow)
        self.horizontalLayout.setObjectName(_fromUtf8("horizontalLayout"))
        self.StatesOverview = QtGui.QVBoxLayout()
        self.StatesOverview.setSpacing(0)
        self.StatesOverview.setSizeConstraint(QtGui.QLayout.SetDefaultConstraint)
        self.StatesOverview.setContentsMargins(0, -1, -1, -1)
        self.StatesOverview.setObjectName(_fromUtf8("StatesOverview"))
        self.StatesDisplay = QtGui.QVBoxLayout()
        self.StatesDisplay.setObjectName(_fromUtf8("StatesDisplay"))
        self.list_state1_label = QtGui.QLabel(MainWindow)
        font = QtGui.QFont()
        font.setPointSize(14)
        self.list_state1_label.setFont(font)
        self.list_state1_label.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.list_state1_label.setFrameShape(QtGui.QFrame.Panel)
        self.list_state1_label.setFrameShadow(QtGui.QFrame.Plain)
        self.list_state1_label.setTextFormat(QtCore.Qt.AutoText)
        self.list_state1_label.setAlignment(QtCore.Qt.AlignCenter)
        self.list_state1_label.setMargin(8)
        self.list_state1_label.setObjectName(_fromUtf8("list_state1_label"))
        self.StatesDisplay.addWidget(self.list_state1_label)
        self.list_state2_label = QtGui.QLabel(MainWindow)
        font = QtGui.QFont()
        font.setPointSize(14)
        self.list_state2_label.setFont(font)
        self.list_state2_label.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.list_state2_label.setFrameShape(QtGui.QFrame.Panel)
        self.list_state2_label.setTextFormat(QtCore.Qt.AutoText)
        self.list_state2_label.setAlignment(QtCore.Qt.AlignCenter)
        self.list_state2_label.setMargin(8)
        self.list_state2_label.setObjectName(_fromUtf8("list_state2_label"))
        self.StatesDisplay.addWidget(self.list_state2_label)
        self.list_state3_label = QtGui.QLabel(MainWindow)
        self.list_state3_label.setEnabled(True)
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Preferred, QtGui.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.list_state3_label.sizePolicy().hasHeightForWidth())
        self.list_state3_label.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(14)
        self.list_state3_label.setFont(font)
        self.list_state3_label.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.list_state3_label.setFrameShape(QtGui.QFrame.Panel)
        self.list_state3_label.setTextFormat(QtCore.Qt.AutoText)
        self.list_state3_label.setScaledContents(False)
        self.list_state3_label.setAlignment(QtCore.Qt.AlignCenter)
        self.list_state3_label.setMargin(8)
        self.list_state3_label.setObjectName(_fromUtf8("list_state3_label"))
        self.StatesDisplay.addWidget(self.list_state3_label)
        self.list_state4_label = QtGui.QLabel(MainWindow)
        font = QtGui.QFont()
        font.setPointSize(14)
        self.list_state4_label.setFont(font)
        self.list_state4_label.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.list_state4_label.setFrameShape(QtGui.QFrame.Panel)
        self.list_state4_label.setTextFormat(QtCore.Qt.AutoText)
        self.list_state4_label.setAlignment(QtCore.Qt.AlignCenter)
        self.list_state4_label.setMargin(8)
        self.list_state4_label.setObjectName(_fromUtf8("list_state4_label"))
        self.StatesDisplay.addWidget(self.list_state4_label)
        self.list_state5_label = QtGui.QLabel(MainWindow)
        font = QtGui.QFont()
        font.setPointSize(14)
        self.list_state5_label.setFont(font)
        self.list_state5_label.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.list_state5_label.setFrameShape(QtGui.QFrame.Panel)
        self.list_state5_label.setTextFormat(QtCore.Qt.AutoText)
        self.list_state5_label.setAlignment(QtCore.Qt.AlignCenter)
        self.list_state5_label.setMargin(8)
        self.list_state5_label.setObjectName(_fromUtf8("list_state5_label"))
        self.StatesDisplay.addWidget(self.list_state5_label)
        self.list_state6_label = QtGui.QLabel(MainWindow)
        font = QtGui.QFont()
        font.setPointSize(14)
        self.list_state6_label.setFont(font)
        self.list_state6_label.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.list_state6_label.setFrameShape(QtGui.QFrame.Panel)
        self.list_state6_label.setTextFormat(QtCore.Qt.AutoText)
        self.list_state6_label.setAlignment(QtCore.Qt.AlignCenter)
        self.list_state6_label.setMargin(8)
        self.list_state6_label.setObjectName(_fromUtf8("list_state6_label"))
        self.StatesDisplay.addWidget(self.list_state6_label)
        self.StatesOverview.addLayout(self.StatesDisplay)
        spacerItem = QtGui.QSpacerItem(20, 5, QtGui.QSizePolicy.Minimum, QtGui.QSizePolicy.Fixed)
        self.StatesOverview.addItem(spacerItem)
        self.Robopoly_logo_label = QtGui.QLabel(MainWindow)
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Preferred, QtGui.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.Robopoly_logo_label.sizePolicy().hasHeightForWidth())
        self.Robopoly_logo_label.setSizePolicy(sizePolicy)
        self.Robopoly_logo_label.setMaximumSize(QtCore.QSize(339, 96))
        self.Robopoly_logo_label.setScaledContents(True)
        self.Robopoly_logo_label.setObjectName(_fromUtf8("Robopoly_logo_label"))
        self.StatesOverview.addWidget(self.Robopoly_logo_label)
        spacerItem1 = QtGui.QSpacerItem(20, 5, QtGui.QSizePolicy.Minimum, QtGui.QSizePolicy.Expanding)
        self.StatesOverview.addItem(spacerItem1)
        self.ToolsPanel = QtGui.QHBoxLayout()
        self.ToolsPanel.setSizeConstraint(QtGui.QLayout.SetDefaultConstraint)
        self.ToolsPanel.setObjectName(_fromUtf8("ToolsPanel"))
        self.side_push_restart = QtGui.QPushButton(MainWindow)
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Minimum, QtGui.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.side_push_restart.sizePolicy().hasHeightForWidth())
        self.side_push_restart.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(11)
        self.side_push_restart.setFont(font)
        self.side_push_restart.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.side_push_restart.setObjectName(_fromUtf8("side_push_restart"))
        self.ToolsPanel.addWidget(self.side_push_restart)
        spacerItem2 = QtGui.QSpacerItem(5, 20, QtGui.QSizePolicy.Fixed, QtGui.QSizePolicy.Minimum)
        self.ToolsPanel.addItem(spacerItem2)
        self.side_push_config = QtGui.QPushButton(MainWindow)
        font = QtGui.QFont()
        font.setPointSize(11)
        self.side_push_config.setFont(font)
        self.side_push_config.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.side_push_config.setObjectName(_fromUtf8("side_push_config"))
        self.ToolsPanel.addWidget(self.side_push_config)
        spacerItem3 = QtGui.QSpacerItem(5, 20, QtGui.QSizePolicy.Fixed, QtGui.QSizePolicy.Minimum)
        self.ToolsPanel.addItem(spacerItem3)
        self.side_push_exit = QtGui.QPushButton(MainWindow)
        font = QtGui.QFont()
        font.setPointSize(11)
        self.side_push_exit.setFont(font)
        self.side_push_exit.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.side_push_exit.setObjectName(_fromUtf8("side_push_exit"))
        self.ToolsPanel.addWidget(self.side_push_exit)
        self.StatesOverview.addLayout(self.ToolsPanel)
        self.StatesOverview.setStretch(0, 1)
        self.horizontalLayout.addLayout(self.StatesOverview)
        spacerItem4 = QtGui.QSpacerItem(5, 20, QtGui.QSizePolicy.Fixed, QtGui.QSizePolicy.Minimum)
        self.horizontalLayout.addItem(spacerItem4)
        self.StatesControl = QtGui.QVBoxLayout()
        self.StatesControl.setObjectName(_fromUtf8("StatesControl"))
        self.image_box = QtGui.QLabel(MainWindow)
        font = QtGui.QFont()
        font.setPointSize(24)
        self.image_box.setFont(font)
        self.image_box.setStyleSheet(_fromUtf8("background-color: rgba(0, 0, 0, 100);"))
        self.image_box.setFrameShape(QtGui.QFrame.Box)
        self.image_box.setAlignment(QtCore.Qt.AlignCenter)
        self.image_box.setObjectName(_fromUtf8("image_box"))
        self.StatesControl.addWidget(self.image_box)
        self.stackedWidget = QtGui.QStackedWidget(MainWindow)
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Preferred, QtGui.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.stackedWidget.sizePolicy().hasHeightForWidth())
        self.stackedWidget.setSizePolicy(sizePolicy)
        self.stackedWidget.setMinimumSize(QtCore.QSize(0, 60))
        self.stackedWidget.setMaximumSize(QtCore.QSize(16777215, 60))
        self.stackedWidget.setObjectName(_fromUtf8("stackedWidget"))
        self.page_state1 = QtGui.QWidget()
        self.page_state1.setObjectName(_fromUtf8("page_state1"))
        self.verticalLayout_3 = QtGui.QVBoxLayout(self.page_state1)
        self.verticalLayout_3.setObjectName(_fromUtf8("verticalLayout_3"))
        self.state1_label = QtGui.QLabel(self.page_state1)
        self.state1_label.setAlignment(QtCore.Qt.AlignCenter)
        self.state1_label.setObjectName(_fromUtf8("state1_label"))
        self.verticalLayout_3.addWidget(self.state1_label)
        self.stackedWidget.addWidget(self.page_state1)
        self.page_state2 = QtGui.QWidget()
        self.page_state2.setObjectName(_fromUtf8("page_state2"))
        self.horizontalLayout_2 = QtGui.QHBoxLayout(self.page_state2)
        self.horizontalLayout_2.setObjectName(_fromUtf8("horizontalLayout_2"))
        self.capture_but = QtGui.QPushButton(self.page_state2)
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Expanding, QtGui.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.capture_but.sizePolicy().hasHeightForWidth())
        self.capture_but.setSizePolicy(sizePolicy)
        self.capture_but.setMinimumSize(QtCore.QSize(0, 0))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.capture_but.setFont(font)
        self.capture_but.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.capture_but.setInputMethodHints(QtCore.Qt.ImhNone)
        self.capture_but.setObjectName(_fromUtf8("capture_but"))
        self.horizontalLayout_2.addWidget(self.capture_but)
        self.import_but = QtGui.QPushButton(self.page_state2)
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Expanding, QtGui.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.import_but.sizePolicy().hasHeightForWidth())
        self.import_but.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(11)
        self.import_but.setFont(font)
        self.import_but.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.import_but.setObjectName(_fromUtf8("import_but"))
        self.horizontalLayout_2.addWidget(self.import_but)
        self.stackedWidget.addWidget(self.page_state2)
        self.page_state3 = QtGui.QWidget()
        self.page_state3.setObjectName(_fromUtf8("page_state3"))
        self.verticalLayout_2 = QtGui.QVBoxLayout(self.page_state3)
        self.verticalLayout_2.setObjectName(_fromUtf8("verticalLayout_2"))
        self.canny_thresh1 = QtGui.QSlider(self.page_state3)
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Expanding, QtGui.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.canny_thresh1.sizePolicy().hasHeightForWidth())
        self.canny_thresh1.setSizePolicy(sizePolicy)
        self.canny_thresh1.setStyleSheet(_fromUtf8("color: rgb(255, 255, 255);"))
        self.canny_thresh1.setMinimum(0)
        self.canny_thresh1.setMaximum(5000)
        self.canny_thresh1.setOrientation(QtCore.Qt.Horizontal)
        self.canny_thresh1.setObjectName(_fromUtf8("canny_thresh1"))
        self.verticalLayout_2.addWidget(self.canny_thresh1)
        self.canny_thresh2 = QtGui.QSlider(self.page_state3)
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Expanding, QtGui.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.canny_thresh2.sizePolicy().hasHeightForWidth())
        self.canny_thresh2.setSizePolicy(sizePolicy)
        self.canny_thresh2.setStyleSheet(_fromUtf8("color: rgb(255, 255, 255);"))
        self.canny_thresh2.setMinimum(0)
        self.canny_thresh2.setMaximum(5000)
        self.canny_thresh2.setProperty("value", 4000)
        self.canny_thresh2.setOrientation(QtCore.Qt.Horizontal)
        self.canny_thresh2.setObjectName(_fromUtf8("canny_thresh2"))
        self.verticalLayout_2.addWidget(self.canny_thresh2)
        self.stackedWidget.addWidget(self.page_state3)
        self.page_state4 = QtGui.QWidget()
        self.page_state4.setObjectName(_fromUtf8("page_state4"))
        self.horizontalLayout_3 = QtGui.QHBoxLayout(self.page_state4)
        self.horizontalLayout_3.setObjectName(_fromUtf8("horizontalLayout_3"))
        self.state4_label = QtGui.QLabel(self.page_state4)
        self.state4_label.setAlignment(QtCore.Qt.AlignCenter)
        self.state4_label.setObjectName(_fromUtf8("state4_label"))
        self.horizontalLayout_3.addWidget(self.state4_label)
        self.state4_progress = QtGui.QProgressBar(self.page_state4)
        self.state4_progress.setEnabled(True)
        self.state4_progress.setProperty("value", 0)
        self.state4_progress.setTextVisible(False)
        self.state4_progress.setObjectName(_fromUtf8("state4_progress"))
        self.horizontalLayout_3.addWidget(self.state4_progress)
        self.horizontalLayout_3.setStretch(0, 1)
        self.horizontalLayout_3.setStretch(1, 1)
        self.stackedWidget.addWidget(self.page_state4)
        self.page_state5 = QtGui.QWidget()
        self.page_state5.setObjectName(_fromUtf8("page_state5"))
        self.verticalLayout_5 = QtGui.QVBoxLayout(self.page_state5)
        self.verticalLayout_5.setObjectName(_fromUtf8("verticalLayout_5"))
        self.state5_pause_but = QtGui.QPushButton(self.page_state5)
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Expanding, QtGui.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.state5_pause_but.sizePolicy().hasHeightForWidth())
        self.state5_pause_but.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(11)
        self.state5_pause_but.setFont(font)
        self.state5_pause_but.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.state5_pause_but.setObjectName(_fromUtf8("state5_pause_but"))
        self.verticalLayout_5.addWidget(self.state5_pause_but)
        self.stackedWidget.addWidget(self.page_state5)
        self.page_state6 = QtGui.QWidget()
        self.page_state6.setObjectName(_fromUtf8("page_state6"))
        self.verticalLayout_6 = QtGui.QVBoxLayout(self.page_state6)
        self.verticalLayout_6.setObjectName(_fromUtf8("verticalLayout_6"))
        self.state6_label = QtGui.QLabel(self.page_state6)
        self.state6_label.setAlignment(QtCore.Qt.AlignCenter)
        self.state6_label.setObjectName(_fromUtf8("state6_label"))
        self.verticalLayout_6.addWidget(self.state6_label)
        self.stackedWidget.addWidget(self.page_state6)
        self.StatesControl.addWidget(self.stackedWidget)
        self.StateChange = QtGui.QHBoxLayout()
        self.StateChange.setObjectName(_fromUtf8("StateChange"))
        self.prev_but = QtGui.QPushButton(MainWindow)
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Minimum, QtGui.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.prev_but.sizePolicy().hasHeightForWidth())
        self.prev_but.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(11)
        self.prev_but.setFont(font)
        self.prev_but.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.prev_but.setObjectName(_fromUtf8("prev_but"))
        self.StateChange.addWidget(self.prev_but)
        self.next_but = QtGui.QPushButton(MainWindow)
        font = QtGui.QFont()
        font.setPointSize(11)
        self.next_but.setFont(font)
        self.next_but.setStyleSheet(_fromUtf8("background-color: rgb(255, 255, 255);"))
        self.next_but.setObjectName(_fromUtf8("next_but"))
        self.StateChange.addWidget(self.next_but)
        self.StatesControl.addLayout(self.StateChange)
        self.StatesControl.setStretch(0, 1)
        self.horizontalLayout.addLayout(self.StatesControl)

        self.retranslateUi(MainWindow)
        self.stackedWidget.setCurrentIndex(0)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        self.list_state1_label.setText(QtGui.QApplication.translate("MainWindow", "<html><head/><body><p>State 1: <br/>Start</p></body></html>", None, QtGui.QApplication.UnicodeUTF8))
        self.list_state2_label.setText(QtGui.QApplication.translate("MainWindow", "<html><head/><body><p>State 2: <br/>Capture</p></body></html>", None, QtGui.QApplication.UnicodeUTF8))
        self.list_state3_label.setText(QtGui.QApplication.translate("MainWindow", "<html><head/><body><p>State 3: <br/>Countour detection</p></body></html>", None, QtGui.QApplication.UnicodeUTF8))
        self.list_state4_label.setText(QtGui.QApplication.translate("MainWindow", "<html><head/><body><p>State 4: <br/>Path computation</p></body></html>", None, QtGui.QApplication.UnicodeUTF8))
        self.list_state5_label.setText(QtGui.QApplication.translate("MainWindow", "<html><head/><body><p>State 5: <br/>Draw</p></body></html>", None, QtGui.QApplication.UnicodeUTF8))
        self.list_state6_label.setText(QtGui.QApplication.translate("MainWindow", "<html><head/><body><p>State 6: <br/>Finish</p></body></html>", None, QtGui.QApplication.UnicodeUTF8))
        self.Robopoly_logo_label.setText(QtGui.QApplication.translate("MainWindow", "logo", None, QtGui.QApplication.UnicodeUTF8))
        self.side_push_restart.setText(QtGui.QApplication.translate("MainWindow", "Restart", None, QtGui.QApplication.UnicodeUTF8))
        self.side_push_config.setText(QtGui.QApplication.translate("MainWindow", "Configuration", None, QtGui.QApplication.UnicodeUTF8))
        self.side_push_exit.setText(QtGui.QApplication.translate("MainWindow", "Exit", None, QtGui.QApplication.UnicodeUTF8))
        self.image_box.setText(QtGui.QApplication.translate("MainWindow", "Image Box", None, QtGui.QApplication.UnicodeUTF8))
        self.state1_label.setText(QtGui.QApplication.translate("MainWindow", "<html><head/><body><p><span style=\" font-size:14pt;\">Welcome!</span></p></body></html>", None, QtGui.QApplication.UnicodeUTF8))
        self.capture_but.setText(QtGui.QApplication.translate("MainWindow", "Capture", None, QtGui.QApplication.UnicodeUTF8))
        self.import_but.setText(QtGui.QApplication.translate("MainWindow", "Import", None, QtGui.QApplication.UnicodeUTF8))
        self.state4_label.setText(QtGui.QApplication.translate("MainWindow", "<html><head/><body><p><span style=\" font-size:14pt;\">X contours found</span></p></body></html>", None, QtGui.QApplication.UnicodeUTF8))
        self.state5_pause_but.setText(QtGui.QApplication.translate("MainWindow", "Pause drawing", None, QtGui.QApplication.UnicodeUTF8))
        self.state6_label.setText(QtGui.QApplication.translate("MainWindow", "<html><head/><body><p><span style=\" font-size:14pt;\">Finished!</span></p></body></html>", None, QtGui.QApplication.UnicodeUTF8))
        self.prev_but.setText(QtGui.QApplication.translate("MainWindow", "Previous", None, QtGui.QApplication.UnicodeUTF8))
        self.next_but.setText(QtGui.QApplication.translate("MainWindow", "Next", None, QtGui.QApplication.UnicodeUTF8))

